waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/client/MetastoreUnavailableExceptionTest.java

```diff
@@ -0,0 +1,26 @@
+package com.hotels.bdp.waggledance.client;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+
+public class MetastoreUnavailableExceptionTest {
+
+  @Test
+  public void messageError() {
+    MetastoreUnavailableException exception = new MetastoreUnavailableException("error");
```

Comment:
Same as my comment on the other test about putting the value into a string that you can assert later instead of declaring it twice.